juju/juju
# Stop but don't cleanup manual machines

## Description of change

When removing manual machines, we stop the jujud agent on the machine but do not clean up or remove any Juju files. And for controllers, we retain mongo etc. We will add a separate command juju clean-juju to do the cleanup explicitly.

## QA steps

bootstrap manual controller
add manual machine
juju remove-machine
ssh into the machine and see that jujud has stopped but /var/lib/juju still there
juju destroy-controller
ssh into the machine and see that jujud has stopped but /var/lib/juju still there
